src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -261,21 +270,23 @@ void AbstractTableGenerator::generate_and_store() {
   {
     std::cout << "- Adding tables to StorageManager and generating table statistics" << std::endl;
     auto& storage_manager = Hyrise::get().storage_manager;
-    auto threads = std::vector<std::thread>{};
+    auto jobs = std::vector<std::shared_ptr<AbstractTask>>{};
+    jobs.reserve(table_info_by_name.size());
     for (auto& table_info_by_name_pair : table_info_by_name) {
       const auto& table_name = table_info_by_name_pair.first;
       auto& table_info = table_info_by_name_pair.second;
 
-      threads.emplace_back([&] {
+      const auto add_table_with_statistics = [&]() {
```

Comment:
Could this be misunderstood that only tables with statistics are added? How about `add_table_build_statistics` or simply `add_table`?